cmd/package-builder/package-builder.go

```diff
@@ -79,6 +79,11 @@ func runMake(args []string) error {
 			env.String("LAUNCHER_VERSION", "stable"),
 			"What TUF channel to download launcher from. Supports filesystem paths",
 		)
+		flLauncherArmVersion = flagset.String(
+			"launcher_arm_version",
+			env.String("LAUNCHER_ARM_VERSION", "stable"),
+			"What TUF channel to download launcher from for ARM. Supports filesystem paths",
+		)
```

Comment:
I added this just so that I could specify a local arm build to package. I can't imagine a situation where we would want the arm and amd binaries to have different code.

This has me thinking though that maybe we should add in some safe guards to ensure the arm and amd versions are the same. If we some how managed to get into a situation where TUF updated one and not the other, we could end up with a package different arches are on different versions.

Reply:
I can't find a coherent way to verify the versions are the same. However, I set up the flag so that it's default is an empty string and it will just set that flag to what ever `launcher_version` is if no `launcher_arm_version` is provided.

Reply:
I'm not sure there's a good way to validate a launcher version, without invoking it. Maaaybe we can string parse it, but I'm not sure it's worth it. (And given what Zack recently learned about file versions and MSIs, let's not go there...)

Maybe this is an argument for being _less_ clever. Command line either takes `launcher_version=` and downloads from TUF (we should have enough metadata there to validate that `stable` is the same) Or `use_local_binary=...` and it ignores TUF

Reply:
I guess that's kinda what would happen. So maybe I'm over thinking it. Maybe we validate the versions from tuf, and skip that validation on local paths.

Reply:
invoking it could be troublesome since we'll have 2 binaries with to different arches, An arm64 machine will probably emulate the amd64 binary and run it, but the reverse doesn't work.

I think just having separate flags for paths is probably good with only a single option for `launcher_version` which is downloaded form tuf for both arches.